get-thru-todo/src/Components/HomeBody/homeBody.jsx

```diff
@@ -0,0 +1,48 @@
+import heroImg from "../../assets/hero-img.jpg";
+import ashneerMeme from "../../assets/ashneer-meme.jpg";
+import "../HomeBody/homeBody.css";
+import { Link } from "react-router-dom";
+const Body = () => {
```

Comment:
would suggest to keep the component name same as file name i.e. like "const HomeBody"